third_party/silabs/efr32_sdk.gni

```diff
@@ -547,12 +547,14 @@ template("efr32_sdk") {
       ]
     }
 
-    if (chip_enable_multi_ota_encryption && chip_enable_multi_ota_requestor) {
-      defines += [ "SL_MATTER_ENABLE_OTA_ENCRYPTION=1" ]
```

Comment:
This change is reverting the define to its previous version when this was part for the provision structure refactor.
Can you remove this change?

Reply:
This macro  `SL_MATTER_ENABLE_OTA_ENCRYPTION`  defined  when both multi ota and encryption enabled.
  ```
  if (chip_enable_multi_ota_requestor) {
      if (chip_enable_multi_ota_encryption) {
```